src/main/java/racingcar/persistence/dao/GameResultDao.java

```diff
@@ -0,0 +1,47 @@
+package racingcar.persistence.dao;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
+import org.springframework.stereotype.Repository;
+import racingcar.entity.GameResultEntity;
+
+import java.sql.PreparedStatement;
+import java.util.List;
+
+@Repository
+public class GameResultDao {
+
+    private static final String[] GAME_RESULT_ID = new String[]{"id"};
+
+    private final JdbcTemplate jdbcTemplate;
+
+    @Autowired
+    public GameResultDao(final JdbcTemplate jdbcTemplate) {
+        this.jdbcTemplate = jdbcTemplate;
+    }
+
+    public Number save(final String winners, final int trialCount) {
+        final String sql = "INSERT INTO GAME_RESULT (winners, trial_count) values (?, ?)";
+        KeyHolder keyHolder = new GeneratedKeyHolder();
+        jdbcTemplate.update(connection -> {
+            PreparedStatement preparedStatement = connection.prepareStatement(sql, GAME_RESULT_ID);
+            preparedStatement.setString(1, winners);
+            preparedStatement.setInt(2, trialCount);
+            return preparedStatement;
+        }, keyHolder);
+        return keyHolder.getKey();
```

Comment:
```suggestion
        return keyHolder.getKey().longValue();
```
longValue를 통해 반환값을 Long 타입으로 하는건 어떤가요?
Number를 이용하는 경우는 잘 접해보지 않은것 같아서 조금 어색하네요

Reply:
감사합니다! 변경했습니다 :)